doc/api/process.md

```diff
@@ -1437,8 +1437,12 @@ tarball.
   architecture and version of the current release. This file is used for
   compiling Node.js native add-ons. _This property is only present on Windows
   builds of Node.js and will be missing on all other platforms._
-* `lts` {string} a string label identifying the [LTS][] label for this release.
-  If the Node.js release is not an LTS release, this will be `undefined`.
+  * `lts` {string} a string label identifying the [LTS][] label for this release.
```

Comment:
is there an indent change here or is just github rendering weirdly?

Reply:
Yep, looks like an indent.